packages/vscode-extension/src/commands/ImportFiles.ts

```diff
@@ -45,6 +45,7 @@ export default class ImportFiles extends BaseAction {
 		}
 
 		if (!selectedFilesPaths) {
+			this.messageService.showInformationMessage(this.translationService.getMessage(IMPORT_FILES.PROCESS_CANCELED));
```

Comment:
We will need to review this with the team, to know if we want to show a "cancelation info message" or nothing when the user hits ESCAPE while being asked from the command palette. 
Same for line 95.